lib/src/extension/register_extension.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:injector/injector.dart';
+import 'package:injector/src/factory/factory.dart';
+
+extension RegisterExtension on Injector {
```

Comment:
I don't understand why we need to create extensions for basic functionality.
I would prefer to add this directly to the injector

Reply:
I second this point of view

Reply:
move it into the injector ✅